File: src/apps/Andriod/app/src/main/java/com/dkellycollins/triggerfinger/data/config/impl/FontConfig.java
package com.dkellycollins.triggerfinger.data.config.impl;

import android.content.Context;

import com.dkellycollins.triggerfinger.R;
import com.dkellycollins.triggerfinger.data.config.IFontConfig;

public class FontConfig extends RConfig implements IFontConfig {

    public FontConfig(Context context) {
        super(context);
    }

    @Override
    public String getPath() {
        return _context.getResources().getString(R.string.font_path);
    }

    @Override
    public int getSize() {
        return _context.getResources().getInteger(R.integer.font_size);
    }

    @Override
    public int getColor() {
        return _context.getResources().getColor(R.color.font_color);
    }
}

File: src/apps/Android-Kotlin/app/src/main/kotlin/com/dkellycollins/triggerfinger/data/config/impl/FontConfig.kt
package com.dkellycollins.triggerfinger.data.config.impl

import android.content.Context

import com.dkellycollins.triggerfinger.R
import com.dkellycollins.triggerfinger.data.config.IFontConfig

class FontConfig(context: Context) : RConfig(context), IFontConfig {

    override val path: String
        get() = _context.resources.getString(R.string.font_path)

    override val size: Int
        get() = _context.resources.getInteger(R.integer.font_size)

    override val color: Int
        get() = _context.resources.getColor(R.color.font_color)
}
